python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -306,15 +306,15 @@ def makeMinimalSchema(filterNameList, addFluxSigma=False,
                 field = "%s_flux" % (filterName,),
                 type = numpy.float64,
                 doc = "flux in filter %s" % (filterName,),
-                units = "?",
+                units = "count",
             )
         if addFluxSigma:
             for filterName in filterNameList:
                 schema.addField(
                     field = "%s_fluxSigma" % (filterName,),
                     type = numpy.float64,
                     doc = "flux uncertainty in filter %s" % (filterName,),
-                    units = "?",
+                    units = "count",
```

Comment:
The units are Janskys, as per http://lsst-web.ncsa.illinois.edu/doxygen/x_masterDoxyDoc/classlsst_1_1meas_1_1algorithms_1_1load_reference_objects_1_1_load_reference_objects_task.html#meas_algorithms_loadReferenceObjects_Schema